aws-rhel8-quay.json

```
@@ -4,11 +4,11 @@
     "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
     "username": "{{env `USERNAME`}}",
     "region": "{{env `AWS_DEFAULT_REGION`}}",
-    "ami_description": "Quay Registry for OCP4",
-    "purpose": "Quay Registry for OCP4",
+    "ami_description": "Quay Registry for OpenShift {{env `OCP_MAJ_VER`}}",
+    "purpose": "Quay Registry for OpenShift {{env `OCP_MAJ_VER`}}",
```

We should clarify the description to indicate this is an OCP content AMI and registry, something like:

```suggestion
    "ami_description": "OpenShift {{env `OCP_MAJ_VER`}} content and registry",
    "purpose": "OpenShift {{env `OCP_MAJ_VER`}} content and registry",
```

Although I'm not sure the ami_description variable here is used, so it could be omitted.